<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="email_input">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#email_input{
border : 1px solid #b1afaf;
border-radius : 10%;
font: 18pt &quot;Century Gothic&quot;;
}
#email_input:focus{
border : 1px solid #df2569;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_input">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pass_input{
border : 1px solid #b1afaf;
border-radius : 10%;
font: 18pt &quot;Century Gothic&quot;;
}
#pass_input:focus{
border: 1px solid #df2569;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;IRANSans&quot;;</string>
   </property>
   <property name="text">
    <string>ایمیل</string>
   </property>
  </widget>
  <widget class="QPushButton" name="operator_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1020</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#operator_btn{
	backgorund-color : transparent;
	border : none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;B narm&quot;;</string>
   </property>
   <property name="text">
    <string>ورود به حساب کاربری</string>
   </property>
  </widget>
  <widget class="QLabel" name="operator_lbl">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1020</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Tools/imgs/operatorLogin.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;IRANSans&quot;;</string>
   </property>
   <property name="text">
    <string>رمز عبور</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#login_btn{
border-radius : 10%;
background-color: #df2569;
color : #fff;
font: 17pt &quot;IRANSans&quot;;
}</string>
   </property>
   <property name="text">
    <string>ورود</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
